pkg/controller/backup_session.go

```diff
@@ -80,12 +81,19 @@ func (c *StashController) runBackupSessionProcessor(key string) error {
 		backupSession := obj.(*api_v1beta1.BackupSession)
```

Comment:
don't need else
`return c.setBackupSessionRunning(backupSession)` at the end

Reply:
fixed